ui/admin/tests/integration/components/form/field/list-wrapper-test.js

```diff
@@ -106,4 +106,28 @@ module('Integration | Component | list-wrapper', function (hooks) {
 
     assert.dom('tbody tr').exists({ count: 3 });
   });
+
+  test('it renders multiple options with a single select field', async function (assert) {
```

Comment:
Could we also get some tests to verify we can add options and remove them?

Reply:
@cameronperera yes absolutely

Reply:
@cameronperera take a look at the new test cases and let me know what you think